app/assets/javascripts/discourse/app/templates/components/bootstrap-mode-notice.hbs

```diff
@@ -0,0 +1,15 @@
+{{#unless this.hidden}}
```

Comment:
I wonder if it's a bit more elegant to 

- move this to the application controller so the component is never rendered empty
- switch `hidden` to something like `showBootstrapModeNotice` 

I believe you used a similar pattern in the "Welcome Topic" banner PR.

Reply:
Makes sense, made the required changes.